static/app/views/performance/transactionDetails/quickTraceMeta.tsx

```diff
@@ -43,58 +46,108 @@ export default function QuickTraceMeta({
   event,
   location,
   organization,
-  quickTrace: {isLoading, error, trace, type},
+  quickTrace,
   traceMeta,
   anchor,
   errorDest,
   transactionDest,
+  project,
 }: Props) {
+  const features = ['performance-view'];
+
+  const noFeatureMessage = t('Requires performance monitoring.');
+
+  const docsLink = getConfigureTracingDocsLink(project?.platform ?? null);
```

Comment:
Could simplify to make it accept project OR platform directly and discriminate in ts, so you don't have all these guards everywhere.

Reply:
Simplified to just take a `AvatarProject` since this is based on the project's platform, passing the project itself makes more sense.